### ISSUE TITLE
Question about the validation dataset

### ISSUE THREAD

**Issue by yjump** [Sue-syx/NAS-for-medical-image-segmentation#1]
Good Job. I come across your repo and find very useful. Here I am questioning about the validation dataset. The validation dateset is the same with training dataset only with different transform? I am doubting that this may lead to some issues.

**Comment by Sue-syx** [comment]
Thanks for your interest. I separated one dataset in proportion into the training set and the validation set. You can find this in data_loader.py

**Comment by yjump** [comment]
Thanks for your kind answer. Sorry for my carelessness.
